package.json

```diff
@@ -1,48 +0,0 @@
-{
-  "name": "@caerostris/parallel-transform-stream",
-  "version": "0.0.1",
-  "description": "Transform streams in parallel",
-  "main": "dist/parallel-transform.js",
-  "scripts": {
-    "build": "babel ./src --out-dir ./dist",
```

Comment:
Test cases do not pass on Node < v4. It won't run on 0.12, unless my test setup is broken. It certainly won't run on v0.10.